contrib/python/podman/Makefile

```diff
@@ -2,20 +2,30 @@ PYTHON ?= /usr/bin/python3
 
 .PHONY: python-podman
 python-podman:
-	$(PYTHON) setup.py bdist
+	$(PYTHON) setup.py sdist bdist
+
+.PHONY: lint
+lint:
+	$(PYTHON) -m pylint podman
```

Comment:
Just a heads-up:  @edsantiago and I have had almost nothing but headaches with pylint.  It "works", just a PITA to maintain for CI:  There are sometimes incompatible variations in message-codes/behaviors across different distros (ubuntu vs RHEL vs Fedora).  Might be worth looking for a different tool in the near-term.

Reply:
@cevich @edsantiago Agreed pylint is very picky.  None of the other "linters" actually lock down the exact version of python as well.  kwargs to methods being the big win with pylint. As a compromise how about a pylint target and have the lint target use yapf or pyflakes?  Something like clobber vs. clean.